--- python-sphinx-1.6.2/sphinx-1.6.2/sphinx/writers/latex.py.babel	2017-06-17 17:55:29.769710524 +0200
+++ python-sphinx-1.6.2/sphinx-1.6.2/sphinx/writers/latex.py	2017-06-17 17:55:52.202185468 +0200
@@ -67,7 +67,7 @@ DEFAULT_SETTINGS = {
     'fontenc':         '\\usepackage[T1]{fontenc}',
     'amsmath':         '\\usepackage{amsmath,amssymb,amstext}',
     'multilingual':    '',
-    'babel':           '\\usepackage{babel}',
+    'babel':           '\\usepackage[english]{babel}',
     'polyglossia':     '',
     'fontpkg':         '\\usepackage{times}',
     'fncychap':        '\\usepackage[Bjarne]{fncychap}',
