diff -Naur Sphinx-1.2.2.orig/sphinx/writers/latex.py Sphinx-1.2.2/sphinx/writers/latex.py
--- Sphinx-1.2.2.orig/sphinx/writers/latex.py	2014-04-16 18:28:15.468189400 +0400
+++ Sphinx-1.2.2/sphinx/writers/latex.py	2014-04-16 18:28:23.962188970 +0400
@@ -147,7 +147,7 @@
         'utf8extra':       '\\DeclareUnicodeCharacter{00A0}{\\nobreakspace}',
         'cmappkg':         '\\usepackage{cmap}',
         'fontenc':         '\\usepackage[T1]{fontenc}',
-        'babel':           '\\usepackage{babel}',
+        'babel':           '\\usepackage[english]{babel}',
         'fontpkg':         '\\usepackage{times}',
         'fncychap':        '\\usepackage[Bjarne]{fncychap}',
         'longtable':       '\\usepackage{longtable}',
