--- python-sphinx-1.6.2/sphinx-1.6.2/doc/Makefile.mantarget	2017-06-17 17:53:58.205853774 +0200
+++ python-sphinx-1.6.2/sphinx-1.6.2/doc/Makefile	2017-06-17 17:54:40.938853495 +0200
@@ -14,6 +14,12 @@ help:
 
 .PHONY: help Makefile
 
+# Likewise for man, since a man directory exist and make will assume
+# the target is up to date
+.PHONY: man
+man:
+	@$(SPHINXBUILD) -M man "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
